telco-core/telco-hub/configuration/reference-crs/required/talm/talmSubscription.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-topology-aware-lifecycle-manager-subscription
+  namespace: openshift-operators
+spec:
+  channel: stable
+  installPlanApproval: Manual
+  name: topology-aware-lifecycle-manager
+  source: redhat-operators
```

Comment:
We expect the primary use case to be in a disconnected environment with a custom catalog to hold only specific versions of operators. Can you make this redhat-operators-disconnected to reflect that?

Reply:
Is TALM in the required list? Core clusters are not always disconnected.